### PR TITLE
Feat/prev tours

### LINKED ISSUE #26: UI MVP: Previous Tours (view all)

### PR DESCRIPTION
**Pull request by rachelbuchta**
### What Type of PR Is This?
- [X] New Feature 
- [ ] Bug Fix
- [ ] Testing
- [ ] Refactoring/Clean Up 

#### What does this branch do? 

* Adds mock data for past tours and past tour details
* Imports data for past tours into PastTours component and renders each tour with location and date
* Adds a search bar in SearchBar component and search functionality by tour location
* Adds a little styling to cards
* Set up tour details page by importing mock data and rendering the inputs for each field so that a user can look back at past tours with the details associated with that tour.
* Imported some icons for details page 

#### Have you tested your branch? How?

* only locally by seeing what shows up on the DOM and watching the console for errors 

#### Do all existing unit and integration tests pass? 

N/A

#### Does this branch require special installation steps? 

N/A

#### Related Issues:

* May have to do some more work as far as how props are being passed down to the details page since I think we will be grabbing those by tour ID and user ID but haven't gotten that far yet
* I think it will make more since when we have router set up and the API
* Definitely needs some more styling, but this is a great start!

closes #26 #27 

#### Screenshots:
Screen Shot 2021-04-11 at 11 36 48 AM
Screen Shot 2021-04-11 at 11 36 28 AM

### CODE REVIEW

File: src/components/PastTours/PastTourDetails.tsx [hunk 1]
```diff
@@ -1,9 +1,94 @@
 import React from 'react'
+import './PastTours.css'
+import question from '../../assets/question-sign.svg'
+import route from '../../assets/travel.svg'
+import lightbulb from '../../assets/light-bulb (1).svg'
```
Comment (deadbelly) [inline src/components/PastTours/PastTourDetails.tsx:6]: I really like these assets. When we settle on our final ui for the form/past tours we should make sure that they look similar.

File: src/components/PastTours/PastTourDetails.tsx [hunk 2]
```diff
@@ -1,9 +1,94 @@
...
+  hazardAvalanche: string;
+  hazardSummary: string;
+  routePreview: string;
+  routeAlternative: string;
+  emergencyPlan: string;
+  rideObservations: string;
+  debriefConditions: string;
+  debriefDecisions: string;
+  debriefPlan: string;
+}
```
Comment (deadbelly) [inline src/components/PastTours/PastTourDetails.tsx:20]: Don't need to do this now but we can maybe have the form and PastTourDetails share an interface

**Review by deadbelly — APPROVED** [review]
This looks great! I really like the icons you used, and want to add them to the form. I also think we can probably refactor to have the form and PastTourDetails share some code down the road which is good.

File: src/components/PastTours/PastTourDetails.tsx (hunk 2 above)
Comment (rachelbuchta) [reply src/components/PastTours/PastTourDetails.tsx:20]: Yeah I was thinking that too! Definitely a ton of refactoring to be done

File: src/components/PastTours/PastTourDetails.tsx (hunk 1 above)
Comment (rachelbuchta) [reply src/components/PastTours/PastTourDetails.tsx:6]: Cool! I was trying to find icons that are similar to the ones in the actual form in the book

Comment (tashiad) [reply src/components/PastTours/PastTourDetails.tsx:6]: Agree! They look great.

File: src/components/PastTours/SearchBar.tsx [hunk 3]
```diff
@@ -1,10 +1,37 @@
...
-      
-    </div>
+    <form className='search-input'>
+      <input
+        type='text'
+        name='input'
+        placeholder='Search By Location'
+        value={input}
+        onChange={(event) => handleChange(event)}
+      />
```
Comment (tashiad) [inline src/components/PastTours/SearchBar.tsx:33]: Made a note of this in the new issue I created to make this work without matching uppercase, but I've done some user testing in the past on mobile search bars and some people get confused when there's not an actual "enter" button next to the input.

**Review by tashiad — APPROVED** [review]
Looks amazing! Such a solid start.

**Pull request merged**